.github/workflows/cleanup-all.yml

```
@@ -45,7 +45,7 @@ jobs:
           go-version: '1.19'
 
       - name: Run cleanup VPC
-        if: always() && github.event.inputs.cleanVPC == 'true'
+        if: github.event_name == 'schedule' && github.event.inputs.cleanVPC == 'true'
```

Does this supposed to be `AND`?
```suggestion
        if: github.event_name == 'schedule' || github.event.inputs.cleanVPC == 'true'
```

---

No

---

Yeah, that's what I thought :)